docs/sdk/core-kit/sfa-react-native/usage.mdx

```diff
@@ -249,7 +249,7 @@ declare type CustomChainConfig = {
   /**
    * Url of the block explorer
    */
-  blockExplorer: string;
+  blockExplorerUrl?: string;
```

Comment:
Hey @Taewa,
Great job with the PR, and thanks for your contribution. But the SFA React native SDK hasn't been updated to use the updated ChainConfig. Once it's done, feel free to raise a PR again. For now, switch this back to blockExplorer. Make these changes and I'll be glad to merge your PR and mark your first contribution here. Thanks!